### DISCUSSION TITLE
是否有内置方法进行打印

### DISCUSSION THREAD

**Discussion by foow123** [antvis/S2#2490] [Q&A]
您好，现在表格导出只能导出为csv，但csv是不带格式的，这样我设计好的表格导出后只能重新排版才能打印，我想请问是否有方法可以按当前表格样式打印表格或者直接导出到excel？

**Comment by lijinke666** [comment]
没有内置的方法, 默认只提供纯文本的导出, 可以借助 https://github.com/SheetJS/sheetjs 或 https://github.com/exceljs/exceljs 等工具自行实现

**Reply by Baili-BL** [reply]
可以支持带格式的导出吗？  我们现在的组件库使用的S2，但最近看到开源的VTable 是支持，建议可以做出这样的案例。感谢https://www.visactor.io/vtable/demo/export/table-export  🙏🙏🙏
